frontend/src/Pages/Control/control.tsx

```diff
@@ -39,6 +39,19 @@ const useStyles = createUseStyles({
   block: {
     flexGrow: "1",
   },
+  logoPreview: {
+    maxWidth: "200px",
+    maxHeight: "200px",
+    overflow: "hidden",
+  },
+  logoPreviewImage: {
+    width: "100%",
+    height: "100%",
+    objectFit: "contain",
+    top: "0",
+    left: "0",
+    margin: "10px",
```

Comment:
Always use % as units as to make sure you don't have more than 100%. Also most common to put margins and padding on divs instead of the img itself